api/app/signals/apps/email_integrations/utils.py

```diff
@@ -126,6 +127,8 @@ def make_email_context(signal: Signal, additional_context: Optional[dict] = None
         'status_state': signal.status.state,
         'handling_message': signal.category_assignment.stored_handling_message,
         'ORGANIZATION_NAME': settings.ORGANIZATION_NAME,
+        'main_category_name': category.parent.name if category.parent else '',
```

Comment:
A Signal is never linked to a parent category, right?

Should we not use the public_name here and have the name as a fallback?
And perhaps the naming could be a bit different, something like `parent_category_name` and `category_name`

Reply:
>A Signal is never linked to a parent category, right?

In practice it should not happen, just defensive programming here.

As to the naming, I want it to be clear what is sub and what is main category. Being vague about it has been a source of misunderstanding before.